release-notes/add-ons/AzureKeyVaultProvider/1.0/1.0.0-preview1.md

```diff
@@ -30,12 +30,14 @@ A sample C# application to demonstrate Always Encrypted with Azure Key Vault can
 
 - Microsoft.Azure.KeyVault 3.0.4
 - Microsoft.Azure.KeyVault.WebKey 3.0.4
+- Microsoft.Data.SqlClient 1.0.19269.1
```

Comment:
Can this be 1.1.0?

Reply:
No, this is correct as per 1.0.0 package dependency tree.